README.md

```diff
@@ -332,7 +338,7 @@ func (w *PCM16Writer) Close() error {
 ...
 
 var writer media.WriteCloser[media.PCM16Sample] = &PCM16Writer{}
```

Comment:
should var type be pointer here?

Reply:
Oops, i also forgot to change the type there

Reply:
Pushed a fix

Reply:
Good to merge?

Reply:
Probably worth looking at making `media` sub-package to split out libopus. Maybe, after this one is merged.

Reply:
That was already done this morning: https://github.com/livekit/server-sdk-go/pull/643